fauxton-visual-guide/index.html

```diff
@@ -0,0 +1,206 @@
+and copy the particular API for the page you are viewing.
+Many times, the browser will shows a interface based on one of the API
+endpoints.</p>
+</div></div></div><div id="page-active-tasks" class="pagecontainer"><div class="leftside page"><img src="assets/active-tasks.png"></div><div class="rightside linen page"><div class="textbox"><h1 id="active-tasks">Active Tasks</h1>
+<p><strong> 1 Change Polling Interval  </strong>
+The status of each task can change as you are watching the Active Tasks
+page. The page auto-updates, but polls at regular intervals. You can set
+the amount of time that the page updates itself by dragging on this bar.</p>
+<p><strong> 2 API URL  </strong>
+This page is an interface to the <a href="http://docs.couchdb.org/en/latest/api/server/common.html#api-server-active-tasks">Active Tasks API</a>
+call. You can view or copy the raw JSON by clicking on this button.</p>
+<p><strong> 3 Filter  </strong><br>You can filter by the type of task, or search active tasks by 
+database name (either source or target) using these tabs.</p>
+</div></div></div><div id="page-config" class="pagecontainer"><div class="leftside page"><img src="assets/config.png"></div><div class="rightside linen page"><div class="textbox"><h1 id="configuration">Configuration</h1>
+<p>An interface into the configuration of your CouchDB installation. The 
+interface allows you to edit the different configurable parameters. For 
+more details on configuration, see the 
+<a href="http://docs.couchdb.org/en/latest/config/index.html#config">Configuring CouchDB</a> 
+section. </p>
+</div></div></div><div id="page-cors" class="pagecontainer"><div class="leftside page"><img src="assets/cors.png"></div><div class="rightside linen page"><div class="textbox"><h1 id="cors">CORS</h1>
+<p>CORS stands for Cross Origin Resource Sharing. It is setting that 
+pertains only to browsers.</p>
+<p>By enabling CORS, you allow your browser to access data from other 
+people, recieving from and sending the data to <i>their</i> computers. 
+For example, you can ping someone else&#39;s CouchDB instance through an 
+AJAX request, and it will return information. Or you can POST data from 
+your browser or site, and enabling CORS allows you to send that 
+information.</p>
+<p>If you disable CORS, that means that no other server outside of your 
+domain that you ping will return information. You can also limit the 
+people you can recieve data from. If you only want certain websites ( 
+origins) to be able to retrieve data from, you can set 
+&#39;Restrict to specific domains&#39;, and then list which origins you want to 
+allow access for.</p>
+<p>By setting, &#39;All domains&#39;, you allow receiving and sending information 
+to anyone.</p>
+<p>This is a browser specific layer of protection, a second layer of 
+protection, if you will. The first layer of protection comes from the 
+permissions you set on each database, and whether they have _reader, 
+_writer, _admin, etc. access.</p>
```

Comment:
I think CouchDB does not know _reader and _writer out of the box, just admins and members.

```
The first layer of protection comes from the permissions you set on each database using roles.
```

and then link http://docs.couchdb.org/en/latest/api/database/security.html